cf-serverd/server_tls.c

```diff
@@ -429,7 +429,7 @@ int ServerTLSSessionEstablish(ServerConnectionState *conn)
         int ret = SSL_accept(ssl);
         if (ret <= 0)
         {
-            TLSLogError(ssl, LOG_LEVEL_ERR,
+            (void)TLSLogError(ssl, LOG_LEVEL_ERR,
```

Comment:
This kind of cast has never been used in all of Core, lets not introduce it now without compelling reason. Did you get any compiler warnings? If so, which compiler?